config/kubernetes-sigs/org.yaml

```
@@ -476,6 +478,7 @@ members:
 - jlamillan
 - jlsong01
 - jmhbnz
+- jmguzik
```

need alpha sorting here to fix the failing job

```suggestion
- jmguzik
- jmhbnz
```

---

commit - 4ad04e37e8c23eceeb878f23b4d23d2a92b064c2

---

Fixed, thanks for pointing out this.